src/main/java/com/znsio/e2e/runner/Runner.java

```diff
@@ -171,10 +179,10 @@ public static String getTestData (String key) {
     }
 
     public static void main (String[] args) {
-        System.out.println("unified-e2e Runner");
-        System.out.println("Provided parameters:");
+        LOGGER.info("unified-e2e Runner");
+        LOGGER.info("Provided parameters:");
         for (int i = 0; i < args.length; i++) {
-            System.out.println("\t" + args[i]);
+            LOGGER.info("\t" + args[i]);
         }
         if (args.length != 3) {
             throw new InvalidTestDataException("Expected following parameters: 'String configFilePath, String stepDefDirName, String featuresDirName, String logDirName");
```

Comment:
Can you remove the earlier incorrect checking - String logDirName?

Reply:
its already changed.